index.html

```diff
@@ -47,9 +47,8 @@
         for title in response.css('h2.entry-title'):
             yield {'title': title.css('a ::text').extract_first()}
 
-        next_page = response.css('div.prev-post > a ::attr(href)').extract_first()
-        if next_page:
-            yield scrapy.Request(response.urljoin(next_page), callback=self.parse)
+        for next_page in response.css('div.prev-post > a'):
+            yield response.follow(next_page, self.parse)
```

Comment:
It looks fine to me, this is the approach I used for Scrapy tutorial.

Probably unrelated, but there is a small difference between old example and new example: if you write

```py
for next_page in response.css('div.prev-post > a ::attr(href)'):
    yield response.follow(next_page, self.parse)
```

(or the whole .extract_first(), etc. as before) `<a>` elements without href are skipped, while with 

```py
for next_page in response.css('div.prev-post > a'):
    yield response.follow(next_page, self.parse)
```

they raise an exception: `<a>` element is already selected, response.follow can't filter it out. Maybe adding a note to Scrapy docs makes sense: using `a` selectors directly is a bit easier and works fine in most case, but it is not 100% the same as using `a::attr(href)`.